includes/wcs-compatibility-functions.php

```diff
@@ -114,10 +114,10 @@ function wcs_get_objects_property( $object, $property, $single = 'single', $defa
 				} elseif ( strtolower( $property ) !== 'id' && metadata_exists( 'post', wcs_get_objects_property( $object, 'id' ), $prefixed_key ) ) {
 					// If we couldn't find a property or function, fallback to using post meta as that's what many __get() methods in WC < 3.0 did.
 					if ( 'single' === $single ) {
-						$value = get_post_meta( wcs_get_objects_property( $object, 'id' ), $prefixed_key, true );
+						$value = $object->get_meta( $prefixed_key, true );
```

Comment:
We don't need to change these as we're already calling `get_meta` if it exists [here](https://github.com/Automattic/woocommerce-subscriptions-core/pull/718/files#diff-5d4ff5a1508ea3b9993736c09e99435b9627ba954aa3658afb445c6e5f53ae87R103-R111)

Reply:
Oh right, reverted it in https://github.com/Automattic/woocommerce-subscriptions-core/pull/718/commits/df0b0e699a1bb1598840bc0f631e53736da10110